web/modules/custom/nidirect_prisons/src/Plugin/rest/resource/PrisonerAmountsResource.php

```diff
@@ -40,8 +39,8 @@ class PrisonerAmountsResource extends ResourceBase implements ContainerFactoryPl
    */
   public function __construct(
     array $configuration,
-    $plugin_id,
-    $plugin_definition,
+          $plugin_id,
```

Comment:
Hard to tell if they are but if they're tabs instead of spaces you'll likely get PHPCS grumbling about them

Reply:
Good spot